openpower/package/pkg-versions.mk

```diff
@@ -0,0 +1,137 @@
+################################################################################
+#
+# pkg-versions
+#
+# Read VERSION.readme in the current directory to learn about the version
+# string structure
+#
+################################################################################
+
+define OPENPOWER_SUBPACKAGE_VERSION
+
+$(2)_VERSION_FILE = $$(OPENPOWER_VERSION_DIR)/$(1).version.txt
+ALL_SUBPACKAGE_VERSIONS += $$($(2)_VERSION_FILE)
+
+### Create subpackage patch file
+define $(2)_OPENPOWER_PATCH_FILE
+
+mkdir -p "$$(OPENPOWER_VERSION_DIR)"
+
+cd "$$(PKGDIR)"; if ls *.patch; then sha512sum *.patch | sha512sum | cut -c 1-7 | \
```

Comment:
I cannot figure out why there are extra double quotes in BR2_GLOBAL_PATCH_DIR
 cd $$(BR2_GLOBAL_PATCH_DIR)/$(1); ls;
and  cd "$$(BR2_GLOBAL_PATCH_DIR)/$(1)"; ls; 
give the below result
cd "" " /esw/san2/smcprek/open-power/forks/op-build/openpower/custom/patches "/hostboot; ls;